src/main/java/exercise_1/Circle.java

```diff
@@ -0,0 +1,51 @@
+package exercise_1;
+
+
+public class Circle {
+    private double radius;
+
+    public Circle() {
+        radius = 1;
```

Comment:
What the hell is this?
Why do you have to set a value in constructor? Why don't you simply pass a value instead?